MekHQ/build.gradle

```diff
@@ -48,15 +48,14 @@ dependencies {
     implementation 'org.apache.commons:commons-csv:1.4'
     implementation 'javax.vecmath:vecmath:1.5.2'
 
-    implementation 'log4j:log4j:1.2.17'
```

Comment:
It's best to piggy-back on the logging libraries from megamek (fetched via transitive dependencies) - this way we won't have versioning conflicts.